src/app/prt-staking/components/prt-section/prt-widget.tsx

```diff
@@ -63,10 +63,15 @@ export function PrtWidget({ token, onClose }: Props) {
     return ''
   }, [currentTab])
 
+  const isTradeButtonDisabled =
+    inputAmount.length === 0 || (currentTab === WidgetTab.STAKE && !canStake)
+
   const onClickTradeButton = useCallback(async () => {
+    if (isTradeButtonDisabled) return
```

Comment:
`disabled` in chakra doesn't actually disable (e.g. button is still clickable): https://github.com/chakra-ui/chakra-ui/issues/7269#issuecomment-2203365511

^ as the thread mentions, we could start setting `isDisabled` on the button, but this would cause a change across the whole app